qutebrowser/completion/models/completionmodel.py

```diff
@@ -154,6 +154,7 @@ def columnCount(self, parent=QModelIndex()):
         """Override QAbstractItemModel::columnCount."""
         # pylint: disable=unused-argument
         return 3
+        # pylint: enable=unused-argument
```

Comment:
No need for this, as pylint already only turns things off for this function and it's needed for the entire function.